constraints/miden-vm/memory.air

```diff
@@ -0,0 +1,101 @@
+mod MemoryChipletAir
+
+### Helper functions/evaluators ###################################################################
+
+fn delta(v: scalar) -> scalar:
+    return v' - v
+
+
+# Enforces that columns must be zero.
+ev is_zero(main: [value[4]]):
```

Comment:
nit: Should this be `values` instead of `value` or is the idea to consider a `word` a single `value`? If so, should we rename this to `word` instead? Wdyt?